dts/bindings/stepper/allegro/allegro,a4979.yml

```
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Allegro A4979 microstepping stepper motor driver.
+  A4979 is a flexible microstepping motor driver with built-in translator for easy operation.
+  It is designed to operate bipolar stepper motors in full-, half-, quarter-, and sixteenth-step
+  modes.
+
+  Example:
+  a4979: a4979 {
+    status = "okay";
```

same  for the other fields :)
```suggestion
      status = "okay";
```